packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcher.tsx

```diff
@@ -14,11 +14,13 @@ const _OrganizationSwitcher = withFloatingTree(() => {
 
   return (
     <Flow.Root flow='organizationSwitcher'>
-      <OrganizationSwitcherTrigger
-        ref={reference}
-        onClick={toggle}
-        isOpen={isOpen}
-      />
+      <Flex>
+        <OrganizationSwitcherTrigger
+          ref={reference}
+          onClick={toggle}
+          isOpen={isOpen}
+        />
+      </Flex>
```

Comment:
Why was this container added?

Reply:
@desiprisg there was an alignment issue. The box where the switcher was contained resembled a `flex-start` situation. But now I see that this is not longer the case. Something outside of it was probably causing the issue. I removed it.

Reply:
@desiprisg @nikosdouvlis actually because of this. You will notice it is not align horizontally 
<img width="610" alt="Screen Shot 2022-11-25 at 2 41 46 PM" src="https://user-images.githubusercontent.com/23478420/203988195-2ddd694a-8995-456b-b62b-525aa431664f.png">

Reply:
Its not clear to me if this is a clerk-js or a dashboard issue